instrumentation/servlet/src/main/java/brave/servlet/ServletRuntime.java

```diff
@@ -4,13 +4,18 @@
 import brave.http.HttpServerHandler;
 import brave.internal.Nullable;
 import java.io.IOException;
+import java.lang.reflect.Method;
+import java.util.LinkedHashMap;
```

Comment:
FYI in case someone is curious.. I tested perf against identityhashmap and a custom size-bounded map. no difference in throughput. chose LinkedHashMap as it almost always the best choice anyway